data/Bradshaw_2022/metadata.yml

```diff
@@ -0,0 +1,175 @@
+source:
+  primary:
+    key: Bradshaw_2022
+    bibtype: Misc
+    year: 2022
+    author: Holly Bradshaw and Greg Guerin
+    title: Variance in plant functional traits in Eucalyptus baxteri and E. obliqua
+      - a pilot study to assess utility for inferring climate vulnerability
+    doi: 10.25909/624BF3FA77AD4
+    publisher: The University of Adelaide
+    keywords: Terrestrial ecology
+    copyright: Creative Commons Attribution 4.0 International
```

Comment:
Should this be license rather than copyright?

Reply:
Changes